web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -347,11 +344,18 @@ class GenerateSupportBundle extends React.Component {
           return;
         }
         const response = await res.json();
+        this.props.updateBundleSlug(response.slug);
         this.setState({ newBundleSlug: response.slug });
         this.state.pollForBundleAnalysisProgress.start(
           this.pollForBundleAnalysisProgress,
           1000
         );
+        this.props.history.push(`/app/${this.props.watch.slug}/troubleshoot`);
+
+        this.setState({
+          isGeneratingBundle: true,
+          generateBundleErrMsg: "",
+        });
```

Comment:
redirect to troubleshoot instead of showing big progress bar. `/troubleshoot` list has progress bar